src/libs/monitoring/scrubbing.ts

```diff
@@ -0,0 +1,12 @@
+import Sentry from '@sentry/react-native'
+
+import { devlog } from '/libs/services/EnvService'
+
+export const scrubPhoneNumbers = (event: Sentry.Event): Sentry.Event => {
+  if (event.message) {
+    event.message = event.message.replace(/tel:[^ ]*/g, 'tel:XXXXXXXXXX')
```

Comment:
What if the tel is in the form `+33 7 12345678`/`07 12 34 56 78`? Or in the US format `(234)-555-1234`?

Reply:
I don't think it matters since we replace everything after "tel:"

Reply:
@Ldoppea I modified the scrubbing so we can have outputs like `tel:(***)-***-****` now